scripts/sign.py

```diff
@@ -67,6 +67,8 @@ def get_args(logger):
         help='Command, one of [' + ', '.join(command_base) + ']')
     parser.add_argument('--uuid', required=True,
                         type=uuid_parse, help='String UUID of the TA')
+    parser.add_argument('--version', type=int_parse, default=0,
+                        help='Decimal version value for the TA, defaults to 0')
```

Comment:
No need to mention "decimal", because the script can parse other bases too (`0xa` works as well as `10`)
I suggest `TA version. 32-bit unsigned integer, defaults to 0`.

Reply:
thanks.
What about this:
"TA version, stored as a 32-bit unsigned integer and used for rollback protection of TA install in the secure database. Defaults to 0."

Reply:
Sounds good to me